plugins/optimization-detective/readme.txt

```diff
@@ -137,6 +137,10 @@ The [plugin source code](https://github.com/WordPress/performance/tree/trunk/plu
 
 == Changelog ==
 
+= 0.1.1 =
+
+* Use plugin slug for generator tag. ([1103](https://github.com/WordPress/performance/pull/1103))
```

Comment:
```suggestion
* Use plugin slug for generator tag. ([1103](https://github.com/WordPress/performance/pull/1103))
* Bump minimum required WP version to 6.4. ([1078](https://github.com/WordPress/performance/pull/1078))
```

Reply:
@mukeshpanchal27 I don't think this is relevant here as the plugin was already released with that bump included, see https://wordpress.org/plugins/optimization-detective/.